python/meterpreter/README.md

```diff
@@ -0,0 +1,7 @@
+## Python Meterpreter
+
+Running unit tests:
+
+```
+python -m unittest discover -v ./tests
```

Comment:
I prefer pytest, but I thought only depending on the inbuilt testing library would be easier for contributors